src/keys.test.js

```diff
@@ -16,7 +16,8 @@ import {
   ExtendedPublicKey,
   fingerprintToFixedLengthHex,
   extendedPublicKeyRootFingerprint,
-  validateExtendedPublicKeyForNetwork
+  validateExtendedPublicKeyForNetwork,
```

Comment:
Are we not properly setup yet for jest+typescript?

Reply:
I don't think so. I forget what the issue was as this was setup a while ago, but only a couple of files are typescript right now anyway.